key-concepts/covenants.adoc

```diff
@@ -176,11 +176,13 @@ millions of blockchains in use.
 
 A sophisticated application need not be monolithic. The Interbit
 platform provides tools so that Interbit applications can request access
-to blockchains via link:chain_joins.adoc[chain joins]. Similar to the
-narrow focus of a covenant, an individual Interbit app often represents
-a single, or narrow concern. By inter-operating with other applications,
-functionality only needs to be implemented once, even if the
-applications are hosted by distinct parties.
+to blockchains via link:chain_joins.adoc[chain joins]. And, since
+Interbit applications are built on link:https://reactjs.org/[React], they
```

Comment:
You can build an Interbit application with anything that will interface with JavaScript. The app need not be React, although we do use React ourselves and give tools to make React development easier with Interbit.